rest.go

```diff
@@ -75,6 +78,33 @@ func getBaseUrl() string {
 	return DEFAULT_BASE_URL
 }
 
+func structToURLValues(input interface{}) string {
```

Comment:
This function returns a query string, I would name it something like `structToQueryString`!

Reply:
we already changed it to Encode in a different internal package as another comment by @tardypad suggested, so it that ok to resolve this?